src/distributions/bernoulli.rs

```diff
@@ -157,6 +157,9 @@ mod test {
 
     #[test]
     fn test_trivial() {
+        // We prefer to be explicit here.
+        #![allow(clippy::bool_assert_comparison)]
```

Comment:
This could be promoted to the lib root instead (since the lint is only about normalisation). But also fine to leave here if you prefer.

Reply:
I think the lint makes sense in other contexts.